pom.xml

```diff
@@ -426,8 +426,14 @@
       </dependency>
       <dependency>
         <groupId>org.glassfish.jersey.media</groupId>
-        <artifactId>jersey-media-json-gson</artifactId>
+        <artifactId>jersey-media-json-jackson</artifactId>
         <version>3.0.12</version>
+        <exclusions>
+          <exclusion>
+            <groupId>com.sun.activation</groupId>
+            <artifactId>jakarta.activation</artifactId>
+          </exclusion>
+        </exclusions>
```

Comment:
I'm surprised by this exclusion.  Are you sure about it?

Reply:
The dependency management duplicate classes check threw an error at build time.

Reply:
I checked in 1fa243b to resolve the conflict but it needs to be tested.